rules/CodeQuality/Rector/Concat/JoinStringConcatRector.php

```diff
@@ -126,6 +126,10 @@ private function joinConcatIfStrings(Concat $node): Concat | String_
         }
 
         $resultString = new String_($leftValue . $rightValue);
+        if (! ctype_print($resultString->value)) {
```

Comment:
symfony/polyfill-ctype may need to be enabled for it 

https://github.com/rectorphp/rector-src/blob/6481ecdb4f118da8e42e1d7123fc21e6fac21a7f/composer.json#L72

Reply:
mb_* functions possibly can be used and available as prefixed vendor symfony/polyfill-mbstring

https://github.com/rectorphp/rector/tree/main/vendor/symfony/polyfill-mbstring

https://stackoverflow.com/questions/6497685/how-do-i-detect-non-ascii-characters-in-a-string

Reply:
it seems can be verified with ascii regex https://github.com/rectorphp/rector-src/pull/3497/commits/bda01fb3e16321e92bd764d40cb9559fec97676c